app/templates/_view_agreements_summary.html

```diff
@@ -0,0 +1,8 @@
+{% if supplier_frameworks %}
+{% set content %}
+  <span class="search-summary-count">{{ supplier_frameworks|length }}</span>
+  {{ pluralize(supplier_frameworks|length, "agreement", "agreements") }}
+  <strong>waiting for countersigning</strong>
```

Comment:
These `<strong>` and `<span>` tags should be `<em>`s to match those used in the buyer app:

https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/1de609c5cb69bd3e1cdc8138f1bd5f78895a86a5/app/presenters/search_summary.py#L10